src/components/CarouselGallery/CarouselGallery.tsx

```diff
@@ -39,7 +39,15 @@ function CarouselGallery({ rotating }: CarouselGalleryProps) {
 			>
 				{PAINTINGS_CYCLE.map(({ image }, i) => (
 					<div key={i} className={styles.painting}>
-						<Image src={image} alt="" fill priority placeholder="blur" />
+						<Image
+							src={image}
+							alt=""
```

Comment:
Is it possible to provide alt text for these images?

Reply:
I think it can be done if `name` key to the objects in `PAINTINGS` and pass those as `alt`.

Reply:
The images on the carousel are repeated and moving, so we consider them to be decorative, but the entire carousel has an `aria-label` description.

Reply:
Alright sounds good